packages/label-sync/src/label-sync.ts

```diff
@@ -114,10 +125,21 @@ export function handler(app: Probot) {
     const owner = c.payload.organization.login;
     const repo = c.payload.repository.name;
 
-    // Allow the label sync logic to be ignored for a repository:
-    const remoteConfiguration = await loadConfig(c);
+    // Allow the label sync logic to be ignored for a repository.
+    // Since there's a feature for ignoring, it should skip the repo
+    // when failed to fetch the config.
+    let remoteConfiguration: ConfigurationOptions | null = null;
+    try {
+      remoteConfiguration = await loadConfig(c);
+    } catch (err) {
+      err.message =
+        'Error reading configuration for ' +
+        `${c.payload.repository.full_name}, skipping: ${err.message}`;
+      logger.error(err);
+      return;
+    }
     if (remoteConfiguration?.ignored) {
-      logger.info(`skipping repository ${repo}`);
+      logger.info(`skipping repository ${c.payload.repository.full_name}`);
```

Comment:
Optional: This logic looks the same as above; could be wrapped in a shared function